sdk/metricsadvisor/ai-metrics-advisor/src/metricsAdvisorKeyCredentialPolicy.ts

```diff
@@ -94,34 +90,16 @@ export class MetricsAdvisorKeyCredential {
  */
 export function createMetricsAdvisorKeyCredentialPolicy(
   credential: MetricsAdvisorKeyCredential
-): RequestPolicyFactory {
+): PipelinePolicy {
   return {
-    create: (nextPolicy: RequestPolicy, options: RequestPolicyOptions) => {
-      return new MetricsAdvisorKeyCredentialPolicy(nextPolicy, options, credential);
+    name: "MetricsAdvisorKeyCredentialPolicy",
```

Comment:
please make the name and the function name as `metricsAdvisorKeyCredentialPolicy`

Reply:
```suggestion
    name: "metricsAdvisorKeyCredentialPolicy",
```